spring-kafka/src/main/java/org/springframework/kafka/config/KafkaStreamsConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.config;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.kafka.streams.StreamsBuilder;
+
+import org.springframework.util.Assert;
+
+/**
+ * Wrapper for {@link StreamsBuilder} properties; used to avoid auto-configuring
+ * a generic {@link Properties} object in Boot.
```

Comment:
I don't think this is a good idea to mentioned Spring Boot here.
More over in its non-official name.

I believe we can find some generic message to say here.

Thanks

Reply:
Good catch on the missed ctor.